Packs/GoogleVault/Integrations/GoogleVault/GoogleVault.py

```diff
@@ -1331,6 +1332,7 @@ def download_export_command():
 def download_and_sanitize_export_results(object_ID, bucket_name, max_results):
     out_file = download_storage_object(object_ID, bucket_name)
     out_file_json = json.loads(xml2json(out_file.getvalue()))
+    out_file.close()
```

Comment:
same here

Reply:
this isn't in a if/else or try/except part

Reply:
then create a try-finally block